app/views/components/_attachment_meta.html.erb

```diff
@@ -29,11 +29,11 @@
 %>
 <%= tag.section class: "gem-c-attachment-meta" do %>
   <% if attachment.reference.present? %>
-    <%= tag.p "Ref: #{attachment.reference}", class: "gem-c-attachment-meta__metadata gem-c-attachment-meta__metadata--compact" %>
+    <%= tag.p "Ref: #{attachment.reference}", class: "gem-c-attachment-meta__metadata govuk-!-margin-bottom-0" %>
```

Comment:
Could we fix this in the main components gem, by using a `last-of-type` selector? It seems like the problem is due to the `> p` applying to all `p` elements, instead of the last one.

Reply:
I’m afraid it looks like it would require a change to govuk-frontend, and then (presumably?) a bump in the govuk-frontend dependency in govuk_publishing_components.
But worse than that, it looks as though the last-of-type suggestion goes against the intended behaviour of the component (i.e. everything should have a 10px margin, except the last child which should have no margin: https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/summary-list/_summary-list.scss#L90-L96)

And if we attempt to fix this in Content Publisher by using stronger selectors (`p.gem-c-attachment-meta__metadata--compact`, to beat the `.govuk-summary-list__value > p`), we get a lint error: `app/assets/stylesheets/components/_attachment-meta.scss:16:1 [W] QualifyingElement: Avoid qualifying class selectors with an element.`

I suspect it's better to use the utility classes provided.